Documentation.md

```diff
@@ -105,8 +105,83 @@ Users have to fill in the fields during the installation process first, on the i
 
 ![Authentication admin](./assets/images/marketplace-configure.png)
 
-## App Settings
-This part is still under development. This is where you will be able to add specific configuration steps such as mapping.
+## App Settings (in progress)
```

Comment:
App Settings (in progress)


Just as in the section dedicated to your app's authentication, you are able to set the parameters that users will need to install your connector.
These are the fields that the iAdvize administrator will fill in to install and configure your connector from the iAdvize Marketplace.

Define your app's settings fields

You can add as many fields as the installation and configuration of your application requires. 
For each of these fields you will have to specify the type of input required.

Label: it is the name of your field. (This is what the users will see).
For instance it could be: Username

Type: it defines the type of entry.
For instance: alphanumeric

ID: this is based on your own code to easily retrieve the different fields. 

These configuration steps will take place immediately after authentication (if any). 
The order of appearance of the steps depends on their order of creation. The first created field will appear first and the last created field will appear last to the user.

Configure your verification Url 
This is the URL on which we will check your app. 
Please read the verification url section right below.